feature-libs/product-configurator/rulebased/components/product-title/configurator-product-title.component.ts

```diff
@@ -45,12 +45,12 @@ export class ConfiguratorProductTitleComponent {
         .pipe(
           map((container) => {
             switch (container.configuration.owner.type) {
-              case CommonConfigurator.OwnerType.PRODUCT:
-                return container.configuration.owner.id;
               case CommonConfigurator.OwnerType.CART_ENTRY:
-                return container.routerData.productCode;
               case CommonConfigurator.OwnerType.ORDER_ENTRY:
-                return container.configuration.overview?.productCode;
+              case CommonConfigurator.OwnerType.SAVED_CART_ENTRY:
+                return container.routerData.productCode;
```

Comment:
For above mentioned configurator types the corresponding `productCode` could be read from the url

Reply:
If we decide to pass `productCode` by clicking on **Configure** button in the catalog or the product detail page, then we could rely on it by fetching `productCode` from router data for `add-to-cart` button if I don't oversee anything

Reply:
No, please see my last remark. Customers might have overwritten our components

Reply:
Okay, I see what you mean.
We could:
1. split rendering of the product title with the corresponding product info from make-to-stock scenario, 
2. define a feature toggle something like `productConfiguratorProductCode` and finally
3. document it according to let a customer know what will happen if this feature toggle will be active
Then a customer could defined whether they could live with it or not

Reply:
The best solution would be to provide a real product code via ccv2 API then customers don't need to think about anything